nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/pom.xml

```diff
@@ -129,6 +168,69 @@ language governing permissions and limitations under the License. -->
                     </excludes>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
```

Comment:
Do we need this section here? I thought we enabled Groovy tests from the top-level POM (although it only happens if there is also at least one Java test). If for some reason it's necessary here, I don't think you need/want to add src/main/groovy as source since the community has decided that non-test code in NiFi must be pure Java.